File: bin/Data/EditorLua/view/ResPreview.lua
local ResPreview = {}
ResPreview.isShow = true
local Translator = require("EditorLua/core/Translate")

function ResPreview:Update()
    if self.isShow == true then
        self.isShow = imgui.Begin(Translator:translate("ResPreview"),self.isShow)
        imgui.End()
    end
    
end

function ResPreview:Show(show)
    self.isShow = show
end

return ResPreview

File: Source/UrhoEditor/views/ResPreview.cpp
#include "ResPreview.h"

namespace Urho3DEditor 
{
ResPreview::ResPreview()
{
}
ResPreview::~ResPreview()
{
}
void ResPreview::Update() 
{
	if (!showing)
		return;
	ImGui::Begin("ResPreview", &showing);
	ImGui::End();

}
}